test/common/upstream/health_checker_impl_test.cc

```diff
@@ -65,27 +58,6 @@ envoy::api::v2::core::HealthCheck createGrpcHealthCheckConfig() {
   return health_check;
 }
 
-TEST(HealthCheckerFactoryTest, createRedis) {
-  std::string json = R"EOF(
-  {
```

Comment:
are there any v1 tests remaining?

Reply:
I think I have moved Redis related tests into [test/extensions/health_checkers/redis/redis_test.cc](https://github.com/envoyproxy/envoy/pull/2993/files?#diff-6102697694e258451a7baf8a3a7eb745) and have it in V2.

Reply:
the v1 api still supported though. so we need to test it.

[cds_json.cc:54-59](https://github.com/envoyproxy/envoy/blob/master/source/common/config/cds_json.cc#L54-L59) is no longer covered: [report](https://38446-65214191-gh.circle-artifacts.com/0/build/envoy/generated/coverage/coverage.source_common_config_cds_json.cc.html)

Reply:
Ah, got it. Will have it then. Thank you for mentioning it.

Reply:
It is covered now. [cds_json.cc:54-59](https://39363-65214191-gh.circle-artifacts.com/0/build/envoy/generated/coverage/coverage.source_common_config_cds_json.cc.html) of cd3398e.